### PR TITLE
bsn/discord notifications

### PR DESCRIPTION
**Pull request by bjornars**
This PR adds a `Makefile` to run common tasks as a yak-shaving measure, but it mainly adds in some github workflows to notify on the Novem discord when there are new PRs and issues.

### CODE REVIEW

File: .github/workflows/discord_pr.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: Discord PR Notification
 
 on:
-  pull_request:
+  pull_request_target:
```
Comment (myme) [inline .github/workflows/discord_pr.yml:4]: This looks sensible.

File: .github/workflows/discord_push.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      commit_url: ${{ github.event.head_commit.url }}
+      repo_full_name: ${{ github.repository }}
+    secrets:
+      discord_webhook: ${{ secrets.DISCORD_WEBHOOK_URL }}
+
+  dummy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Make sure the action is always successful
+        run: echo happy
```
Comment (myme) [inline .github/workflows/discord_push.yml:24]: Why do we need this? It's not there in any of the other ones.

**Review by myme — APPROVED** [review]
LGTM :rocket:

File: .github/workflows/discord_push.yml (hunk 2 above)
Comment (bjornars) [reply .github/workflows/discord_push.yml:24]: This is lifted from the `wiki` repo. I remember there being a problem once where the notification was sent ok, but the workflow "failed" for some unknown reason.

**Pull request merged**